model.py

```diff
@@ -8557,6 +8557,103 @@ def update_credentials(self, _db, access_token, credential):
         _db.commit()
 
 
+class ExternalIntegration(Base):
+
+    """An external integration contains configuration for connecting
+    to a third-party API.
+    """
+
+    __tablename__ = 'externalintegrations'
+    id = Column(Integer, primary_key=True)
+
+    # If there is a special URL to use for access to this API,
+    # put it here. This is most common for OPDS and ODL integrations.
```

Comment:
I think the OPDS/ODL bit isn't true anymore, since that has become the external_account_id for OPDS/ODL integrations.